nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/csv/CSVUtils.java

```diff
@@ -37,6 +45,16 @@
         "The format used by Informix when issuing the UNLOAD TO file_name command with escaping disabled");
     static final AllowableValue MYSQL = new AllowableValue("mysql", "MySQL Format", "CSV data follows the format used by MySQL");
 
+    static final AllowableValue SCHEMA_ACCESS_STRATEGY_EXPLICIT_COLUMNS = new AllowableValue("csv-explicit-columns", "Use '" + EXPLICIT_COLUMNS_DISPLAY_NAME + "' Property",
+            "Takes the '" + EXPLICIT_COLUMNS_DISPLAY_NAME + "' property value as the explicit definition of the CSV columns.");
+
+    static final PropertyDescriptor EXPLICIT_COLUMNS = new PropertyDescriptor.Builder()
+            .name(EXPLICIT_COLUMNS_DISPLAY_NAME)
+            .description("Specifies the CSV columns expected as a comma separated list. Only used with the Schema Access Strategy '" + SCHEMA_ACCESS_STRATEGY_EXPLICIT_COLUMNS.getDisplayName() + "'.")
+            .expressionLanguageSupported(false)
```

Comment:
I didn't immediately see a use for it, but it'd be better to let people use the expression language if it's not going to cause any issues. Will change.